exercise_1-4.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+<html>
+	<head>
+		<meta name="viewport" content="width=device-width, initial-scale=1.0">
+		<meta name="keywords" content="HTML">
+		<meta name="description" content="Sing Up">
+		<meta name="author" content="Juan Cruz Yezzi">
+		<meta charset="utf-8">
+		<title>Topic 0</title>
+	</head>
+	<body align="center">
+		<form name="Sing Up" action="exercise_1-4.html" method="get">
+			<input type="text" name="txtfname" size="30" maxlength="12" value="First Name...">
+			<input type="text" name="txtlname" size="30" maxlength="20" value="Last Name...">
+				<br><br>
+			<input type="text" name="txtemail" size="30" maxlength="35" value="Email...">
+			<input type="text" name="txtbrtd" size="30" maxlength="10" value="Birthday...">
+				<br><br>
+			<select name="Sports" size="3" multiple>
+				<option value="FB">Football</option>
+				<option value="BQ">Basquetball</option>
+				<option value="BB">Baseball</option>
+				<option value="RG">Rugby</option>
+				<option value="TS">Tennis</option>
+				<option value="BX">Boxing</option>
+			</select>
+			<textarea name="Bio" cols="40" rows="3" maxlength="180">Write a small bio...</textarea>
+				<br><br>
+			<input type="Submit" name="Submit" value="Submit">
```

Comment:
I case of a button functionality is better to declare a `button` instead of an `input`